data/transactions/verify/txn.go

```diff
@@ -46,10 +46,6 @@ func TxnPool(s *transactions.SignedTxn, spec transactions.SpecialAddresses, prot
 		return errors.New("empty address")
 	}
 
-	if s.Sig != (crypto.Signature{}) && !s.Msig.Blank() {
-		return errors.New("signedtxn should only have one of Sig or Msig")
-	}
-
```

Comment:
Are these conditions covered elsewhere ?

Reply:
yes, this is covered inside stxnVerifyCore